packages/discord.js/src/structures/InteractionWebhook.js

```diff
@@ -29,7 +29,21 @@ class InteractionWebhook {
   /**
    * Sends a message with this webhook.
    * @param {string|MessagePayload|InteractionReplyOptions} options The content for the reply
-   * @returns {Promise<Message|APIMessage>}
+   * @returns {Promise<Message>}
+   */
+
+  /**
+   * Gets a message that was sent by this webhook.
+   * @param {Snowflake|'@original'} message The id of the message to fetch
+   * @param {WebhookFetchMessageOptions} [options={}] The options to provide to fetch the message.
+   * @returns {Promise<Message>} Returns the message sent by this webhook
+   */
+
+  /**
+   * Edits a message that was sent by this webhook.
+   * @param {MessageResolvable|'@original'} message The message to edit
+   * @param {string|MessagePayload|WebhookEditMessageOptions} options The options to provide
+   * @returns {Promise<Message>} Returns the message edited by this webhook
    */
   send() {}
```

Comment:
Shouldn't this comment be above the `editMessage()` method? And the same goes for the other comments above this one

Reply:
> to override the jsdoc and keep consistency

I understand that, but wouldn't the jsdocs have to be above each method?